engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/SchemaRegistryClient.scala

```diff
@@ -13,21 +13,7 @@ trait SchemaRegistryClient extends Serializable {
     * @param isKey
     * @return
     */
-  def getLatestFreshSchema(topic: String, isKey: Boolean): Validated[SchemaRegistryError, SchemaWithMetadata]
-
-  /**
-    * Latest schema by subject - we assume there can be some latency
-    *
-    * @param topic
-    * @param isKey
-    * @return
-    */
-  def getLatestSchema(topic: String, isKey: Boolean): Validated[SchemaRegistryError, SchemaWithMetadata]
-
-  def getSchema(topic: String, version: Option[Int], isKey: Boolean): Validated[SchemaRegistryError, SchemaWithMetadata] =
```

Comment:
Right now, we don't need it?

Reply:
well - it comiles so I guess we don't need it :)